spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
Sorry for my poor English... :sweat: 

According to [your implementation ](https://github.com/katsuma/itunes-client/pull/22/files#diff-a96b98d7c40c0cacf2df9e63343fce7bR9) you should check the argument of `execute_script` is given `an_instance_of Fixnum` and it is NOT given `Fixnum` class directly.

So I think 

```ruby
let(:level) { an_instance_of(Fixnum) }
...
expect(Volume).to receive(:execute_script).with('volume/up.scpt', level)
```

is better than 

```ruby
let(:level) { Fixnum }
...
expect(Volume).to receive(:execute_script).with('volume/up.scpt', level)
```

So what I say is a simple topic. :eyes: 


By the way,

> do you mean returning the volume value when calling up or down commands?

I think this idea is good. :+1: